tasks/internal/create-advisory-task/create-advisory-task.yaml

```diff
@@ -191,7 +235,6 @@ spec:
               # If after filtering, no images are left, then we can exit early
               if jq -e 'length == 0' <<< "$CONTENT_IMAGES" >/dev/null; then
                   echo "Matching advisory exists: ${ADVISORY_FILE}. Skipping creation."
-                  ADVISORY_URL="${GIT_REPO//\.git/}/-/blob/main/${ADVISORY_FILE}"
```

Comment:
not sure why this is being removed

Reply:
This was leftover from a previous PR. The correct value is on line 197